Makefile

```diff
@@ -18,6 +21,12 @@ copy-crd:
 
 update-all: copy-crd update
 
+verify-crds:
+	$(deps_diff) {vendor/github.com/open-cluster-management/api/cluster/v1,manifests/hub}/0000_00_clusters.open-cluster-management.io_spokeclusters.crd.yaml || ( echo 'crd content is incorrect' && false )
```

Comment:
we need to check all the yamls, right?

Reply:
yes, changes to use script and check all yamls on the dir